src/app/static/src/app/components/Stepper.vue

```diff
@@ -49,6 +36,10 @@
                 <download-results v-if="isActive(7)"></download-results>
             </div>
         </div>
+        <template v-if="activeStep === 3">
```

Comment:
might want to test this logic as well, i.e. that the extra widget is shown on the appropriate step